src/algebraic_geometry/Scheme.lean

```diff
@@ -155,7 +155,7 @@ Spec.LocallyRingedSpace_map_comp f g
 /--
 The spectrum, as a contravariant functor from commutative rings to schemes.
 -/
-@[simps] def Spec : CommRingᵒᵖ ⥤ Scheme :=
```

Comment:
Can you explain why this gets in the way? Is the intent that `Spec_map` and `Spec_obj` are never used directly?

Reply:
If so, can we add that to the docstrings of those declarations?

Reply:
I mean, not having `@[simps]` is the default, isn't it? It's just generally better to not have unnecessary simp lemmas, and to not unnecessarily unfold definitions. I don't think this warrants explanation, or rather the only explanation I would give is that general rule...

Reply:
I guess this answer is not really fair, as the `Spec_obj` lemma was useful in `properties.lean`, to help dsimp something to find an instance. I'm not really sure what to say except that "these simp lemmas caused simp to go down a bad path (once in mathlib3, once in mathlib4 in a different place).

Reply:
I think it is still true that `Spec.obj (op X)` is the simp normal form rather than `Spec_obj X`, and maybe we can record it here if needed?

Reply:
In that case, one compromise would be to use `@[simps {attrs := []}]` or whatever the spelling of "generate the lemmas but don't make them simp" is.

Reply:
A better option would be to add the preferred versions of the lemmas manually, as
```lean
@[simp] lemma Spec_obj_eq : spec_obj X = Spec.obj (op X) := rfl
@[simp] lemma Spec_map_eq : spec_map f = Spec.map (op f) := rfl
```

Reply:
Done.